workshops.qmd

```diff
@@ -58,7 +58,7 @@ From the meeting page, copy the registration link to be included in your worksho
 
 ### Advertise
 
--   Use [this spreadsheet](https://docs.google.com/spreadsheets/d/155tcna1dMWwsZymdKFZPCrnZf3xocw4hPv1JKwOu9MQ/edit#gid=0) to keep track of where you have advertised. 
+-   Use [this spreadsheet](https://docs.google.com/spreadsheets/d/155tcna1dMWwsZymdKFZPCrnZf3xocw4hPv1JKwOu9MQ/edit#gid=0) to keep track of where you have advertised.
```

Comment:
just FYI this is not me being super pedantic—this was the visual markdown editor's change.